wasm/src/main/java/com/dylibso/chicory/wasm/types/Table.java

```diff
@@ -1,12 +1,20 @@
 package com.dylibso.chicory.wasm.types;
 
+import java.util.List;
 import java.util.Objects;
 
 public class Table {
     private final ValueType elementType;
     private final TableLimits limits;
 
     public Table(ValueType elementType, TableLimits limits) {
+        this(
+                elementType,
+                limits,
+                List.of(new Instruction(-1, OpCode.REF_NULL, new long[] {elementType.typeIdx()})));
+    }
+
+    public Table(ValueType elementType, TableLimits limits, List<Instruction> init) {
```

Comment:
`init` is not used in this PR, is it preparation for the follow up?

Reply:
yep! in part 2 this might be the result of a constant expression

Reply:
:+1: